.github/workflows/ci.yml

```diff
@@ -0,0 +1,33 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
We car un this on all push, not only on master push.

Reply:
The problem is, that will duplicate CI builds when you push to a branch and open a PR. Do you want me to drop the branch filter?

Reply:
I was thinking to have this just on push without a branch filter and not for opening PR.

Reply:
Works for me, but then the tests won't be run for PRs coming from other users, like me.

Reply:
Let's go with
```
on:  
  push:
  pull_request:
    branches:
      - master
```

Reply:
Done